package.json

```diff
@@ -30,7 +30,10 @@
     "url": "https://github.com/jstransformers/jstransformer-lodash/blob/master/LICENSE.md"
   },
   "dependencies": {
-    "lodash": "^3.6.0"
+    "lodash.assign": "^3.2.0",
```

Comment:
I'd prefer not to, since we can only guarantee that 3.2.0 or above works. Semver is only about backwards compatibility. 3.2.0 must be compatible to 3.0.0, not the other way around.